CMake/kwiver-setup-tests-python.cmake

```diff
@@ -0,0 +1,29 @@
+#
+# Script to set up environment for python nosetests
+#
+
+
+# results of tests being run will be exported to an Xunit xml file
+if (NOSE_RUNNER)
+
+  set(no_install TRUE)
+  string(TOLOWER "${CMAKE_PROJECT_NAME}" project_name)
+
+  if (WIN32)
+    set(kwiver_test_output_path    "${KWIVER_BINARY_DIR}/$<CONFIG>/bin" )
+  else ()
+    set(kwiver_test_output_path    "${KWIVER_BINARY_DIR}/tests/bin" )
+  endif ()
+  if(VENV_CREATED)
+      if(Python3_INTERPRETER_ID STREQUAL "Anaconda")
+        set(NOSE_COM "$ENV{CONDA_EXE} activate testing_venv && ")
```

Comment:
is NOSE_COM == NOSE_COMMAND? If so, and very minor, I kind of prefer to avoid the abbreviation for clarity.